service/src/main/java/org/databiosphere/workspacedataservice/dao/RecordDao.java

```diff
@@ -983,8 +986,7 @@ private Object getAttributeValueForType(Object object, DataTypeMapping typeMappi
         return getArrayValue(pgArray.getArray(), typeMapping);
       }
       if (typeMapping == DataTypeMapping.JSON) {
-        return objectMapper.readValue(
-            object.toString(), new TypeReference<Map<String, Object>>() {});
+        return objectMapper.readValue(object.toString(), new TypeReference<Object>() {});
```

Comment:
```suggestion
        return objectMapper.readValue(object.toString(), Object.class);
```
I think since you're no longer deserializing into a class with a generic, you don't need to use `TypeReference` anymore which is there to overcome type erasure when deserializing back into generic Java types.